solver/src/solver.rs

```diff
@@ -139,7 +139,7 @@ pub fn create(
     fee_factor: f64,
     min_order_size_one_inch: U256,
     disabled_one_inch_protocols: Vec<String>,
-    paraswap_slippage_bps: usize,
+    paraswap_slippage_bps: u32,
```

Comment:
nanonit: `paraswap_partner_header_value` is now not a very accurate variable name anymore.